requirements.adoc

```diff
@@ -6,7 +6,11 @@ Before you install the multicluster engine for Kubernetes operator 1.0, review t
 * <<supported-operating-systems-and-platforms,Supported operating systems and platforms>>
 * <<network-configuration,Network configuration>>
 
-*Note:* There is no console for the Technical Preview version of this product.
+*Important:* Install multicluster engine for Kubernetes on a cluster that is not a Red Hat Advanced Cluster Management for Kubernetes cluster. The multicluster engine for Kubernetes cannot co-exist with Red Hat Advanced Cluster Management for Kubernetes because multicluster engine for Kubernetes and Red Hat Advanced Cluster Management provide some of the same management components.
```

Comment:
First sentence: "... for Kubernetes coluster." -> "...for Kubernetes hub cluster."  (Add word hub)

Sug:  I think you could shorten the second sentence by saying "...because the two operators provide..." rather than spelling out both product names for a second time in the sentence.